cves/CVE-2014-0117.yml

```diff
@@ -67,12 +72,17 @@ fixes_vcc_instructions: |
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
   - commit: 0ad9ec32504e63c6017600c7de7ec4b4561b897d
-    note: Note that 6fdc17af7e9425c7e7e2b31cc5cf64e961b8c72e is a rollup including this change
-  - commit: 95e0c589d671e450d21a2009098be5a0ff47e800
-    note: Just the STATUS update
+    note: |
+      Note that 6fdc17af7e9425c7e7e2b31cc5cf64e961b8c72e is a rollup including this change.
+      This made sure to include semi-colons along with commas as delineators, not just commas.
+  - commit: bb2749fd6e18dd1682bd1da89b160ca4175c2ec1
+    note: Direct reference to the CVE. Official fix
 vccs:
-  - commit: 11b2dd1d80269fcf75ceac9d962aad7532979bcc
-    note:
+  - commit: d2e6782ea7a
```

Comment:
Please fix this to the full commit has instead of the short one. This should fix the CI